### ISSUE TITLE
Debian Package pulls in GTK 3

### ISSUE THREAD

**Issue by alexbarton** [keybase/client#17990]
Up to version 4.0.0 I was able to use keybase on my server without all of the libraries required for the GUI, namely GTK 3. This has been changed by commit 1f93565cf3ec1c91e5e9 (see #9872 and #17365), which (correctly) pulls in all the dependencies required for working with the GUI.

But I don't use the GUI at all, only the command line … probably time for something like a "*keybase-cli*" package?

The upgrade from 4.0.0-20190508150240.6614a49937 to 4.1.0-20190612201656.952fee6c59 wants to pull in ~84 MB of *new* dependencies – that I don't need at all … not the end of the world, but a bit "ugly", I think …

Any thoughts on this?

**Comment by heronhaye** [comment]
Yeah, we're aware of the issue, especially annoying on servers. One thing you could do is just ignore the dependency warnings / make dummy packages that satisfy the requirements to make apt happy. But that's pretty annoying, so we'll eventually publish these packages. Another temporarily solution would be to build from source yourself. You likely just need `keybase`, `kbfsfuse`, the systemd units, and maybe `run_keybase`.

**Comment by eli-schwartz** [comment]
Note: cli-only packages exist today in Arch Linux. If you could find someone to sponsor keybase for Debian's official repositories that would probably help as I expect Debian would automatically opt for a split package.
